src/platforms/php/guides/laravel/other-versions/laravel5-6-7.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Laravel 5.x, 6.x and 7.x
+description: "Learn about using Sentry with Laravel 5.x, 6.x and 7.x."
+sidebar_order: 600
+redirect_from:
+  - /platforms/php/guides/laravel/other-versions/laravel5-6/
+  - /platforms/php/guides/laravel/configuration/other-versions/laravel5-6/
+---
+
+Laravel is supported via a native package: [sentry-laravel](https://github.com/getsentry/sentry-laravel).
+
+This guide is for Laravel 5.x, 6.x and 7.x. We also provide instructions for [the latest Laravel](/platforms/php/guides/laravel/) as well as [Lumen-specific instructions](/platforms/php/guides/laravel/other-versions/lumen/).
+
+## Install
+
+Install the `sentry/sentry-laravel` package:
+
+```bash
+composer require sentry/sentry-laravel
+```
+
+If you're on Laravel 5.5 or later the package will be auto-discovered. Otherwise you will need to manually configure it in your `config/app.php`.
```

Comment:
```suggestion
If you're on Laravel 5.5 or higher, the package will be auto-discovered. Otherwise you will need to manually configure it in your `config/app.php`.
```